tests/test_reporting.py

```diff
@@ -73,3 +78,76 @@ def test_reporting_stats(self):
 
         # shold not have changed, if still using cached values
         self.assertEqual(len(stats2['encounters']['all']), 5)
+
+    def test_overdue_stats(self):
+        crv = Organization(name='CRV')
+        epic26 = Questionnaire(name='epic26')
+        with SessionScope(db):
+            db.session.add(crv)
+            db.session.add(epic26)
+            db.session.commit()
+        crv, epic26 = map(db.session.merge, (crv, epic26))
+
+        bank = QuestionnaireBank(
+            name='CRV', organization_id=crv.id,
+            start='{"days": 1}',
+            overdue='{"days": 2}',
+            expired='{"days": 90}')
+        qbq = QuestionnaireBankQuestionnaire(
+            questionnaire_id=epic26.id,
+            rank=0)
+        bank.questionnaires.append(qbq)
+
+        self.test_user.organizations.append(crv)
```

Comment:
@ivan-c Is there any way to have codacy ignore the unit test code? Besides false-positive "issues" like the one here, it looks like it may also interpret a lot of "clones" in the tests as well (because we repeat similar steps in different tests, etc).

Reply:
@OptimusRhine Sorry about that- I thought Codacy kept everything separate (no PR comments etc). Let me disable checking the `tests/` dir but please let me know if it's nagging is too much and I'll turn it off.